awx/main/management/commands/graph_jobs.py

```diff
@@ -1,9 +1,14 @@
 # Python
-import asciichartpy as chart
 import collections
 import time
 import sys
 
+try:
+    import asciichartpy as chart
+except:
+    print("Error: this command requires asciichartpy library to be installed")
```

Comment:
Might be defunct already, but...

The `except` should be of an `ImportError`, also I think the message should give a `pip install ...` recommendation, because the _import name_ is often different from the library name, and unless it is spelled out exactly I may feel the need to go look it up. Also, maybe state that it has to be done by root? Into awx venv?